rasa/nlu/utils/spacy_utils.py

```diff
@@ -129,18 +130,96 @@ def get_text(self, example, attribute):
 
         return self.preprocess_text(example.get(attribute))
 
+    @staticmethod
+    def merge_content_lists(
+        indexed_training_samples: List[Tuple[int, Text]], doc_lists: List[Tuple[int, "Doc"]]
+    ) -> List[Tuple[int,"Doc"]]:
+        """Merge lists with processed Docs back into their original order."""
+
+        dct = dict(indexed_training_samples)
+        dct.update(dict(doc_lists))
+        return list(sorted(dct.items()))
+
+    @staticmethod
+    def filter_training_samples_by_content(
+        indexed_training_samples: List[Tuple[int, Text]]
+    ) -> Tuple[List[Tuple[int, str]], List[Tuple[int, str]]]:
```

Comment:
Just to be consistent between `Text` and `str` throughout the code. 
```suggestion
    ) -> Tuple[List[Tuple[int, Text]], List[Tuple[int, Text]]]:
```

Reply:
Fixed in 8ab6e87